### ISSUE TITLE
Support Google AMP

### ISSUE THREAD

**Issue by bogas04** [KhalisFoundation/SikhiToTheMax-Web#375]
This is good for SEO and let's user see content much faster.

**Comment by bogas04** [comment]
Blocked by adding SSR support.

**Comment by saintsoldierx** [comment]
@ManjotS Do you know if this is doable? Not sure what the blocker is but wanting to know whether it can be worked on or still blocked.

**Comment by ManjotS** [comment]
I don't know what SSR is. Somebody with working knowledge of this will need to respond. @bogas04 @navdeepsingh @tsingh777

**Comment by tsingh777** [comment]
SSR is server-side-rendering

**Comment by ManjotS** [comment]
we have server side rendering according to the last conversation I saw on slack.

Is it blocked by having it or not having it?

**Comment by bogas04** [comment]
We have SSR for purpose of rendering the initial HTML chunk, not the actual content. That is where frameworks like NextJS come into place, however for our scenario it's better we add webpack+babel tooling for our server to be able to render react components.

**Comment by ManjotS** [comment]
I think @plug-n-play is working on nextjs

**Comment by plug-n-play** [comment]
Yes @ManjotS - Helping hand(s) would be great. @bogas04 and I have been advocating this since long. Would really appreciate if we can give this a serious shot and move forward. This would help help sttm.co a lot.

**Comment by saintsoldierx** [comment]
Related to #329

**Comment by navdeepsingh** [comment]
@plug-n-play what's the status of Nextjs transformation at your side?

**Comment by plug-n-play** [comment]
I m no longer working on it

**Comment by saintsoldierx** [comment]
@gurjit03 please look into this and feedback.
